api/Cargo.toml

```diff
@@ -21,6 +21,7 @@ chrono = "0.4"
 config = {version = "0.10", default-features = false, features = ["json"]}
 diesel = {version = "^1.4.3", default-features = false, features = ["chrono", "postgres", "r2d2", "uuidv07"]}
 env_logger = "0.7"
+futures = "0.3"
```

Comment:
This is already in our dep tree, we just didn't have to rely on it directly before.